code/__DEFINES/jobs.dm

```diff
@@ -45,3 +45,4 @@
 #define JOB_DISPLAY_ORDER_DETECTIVE 33
 #define JOB_DISPLAY_ORDER_SECURITY_OFFICER 34
 #define JOB_DISPLAY_ORDER_PRISONER 35
+#define JOB_DISPLAY_ORDER_RABBI 36
```

Comment:
This should probably be 16, as it is related to the Chaplain.
Shift the rest up to match.